DataRepo/example_data/obob_sample_table.tsv

```
@@ -1,4 +1,4 @@
-Sample Name	Date Collected	Researcher Name	Tissue	Collection Time	Animal ID	Animal Genotype	Animal Body Weight	Infusate	Infusion Rate	Tracer Concentrations	Animal State	Study Name	Study Description
+Sample Name	Date Collected	Researcher Name	Tissue	Collection Time	Animal ID	Animal Genotype	Animal Body Weight	Infusate	Infusion Rate	Tracer Concentrations	Feeding Status	Study Name	Study Description
```

C. Feeding status had to be changed because the column header didn't match. Since the column is optional, the previous version of the load script silently ignored this scenario. The new load script doesn't allow unknown headers and causes an exception, so the header had to be corrected.